### DISCUSSION TITLE
Play counts and Played status for non-admin users as collection criteria

### DISCUSSION THREAD

**Discussion by benfishbus** [meisnate12/Plex-Meta-Manager#478] [Q&A]
I learned from the Plex Forum that smart collection criteria _plays_, _unplayed_, _progress_, etc. only reflect the admin account's activity - not the activity of the Plex user who is viewing the collection. So I can't focus collections for my Plex Home users on unwatched content. (I know I can still use smart playlists, but nobody wants to see that.)

The Tautulli API has methods for getting lists of users and of plays filtered by user. Can the Tautulli Builders be expanded to address?

**Comment by meisnate12** [comment]
so you want to have a tautulli builder that somehow makes a list of movies a user has played/unplayed/in progress and then your going to make a collection per user?

**Reply by benfishbus** [reply]
Exactly.

**Reply by meisnate12** [reply]
there not an easy way to make this happen from tautulli about the only thing i can think of doing is maybe add a filter that will filter out played movies from a user so you could do any collection and then filter out all the played movies by a specific user but i don't think ill be able to add more functionality then that

**Reply by benfishbus** [reply]
I think that might be enough
